src/Aspire.Hosting.Docker/DockerComposeServiceExtensions.cs

```diff
@@ -48,4 +48,63 @@ public static IResourceBuilder<T> PublishAsDockerComposeService<T>(this IResourc
 
         return builder;
     }
+
+    /// <summary>
+    /// Creates a placeholder for an environment variable in the Docker Compose file.
+    /// </summary>
+    /// <param name="manifestExpressionProvider">The manifest expression provider.</param>
+    /// <param name="dockerComposeService">The Docker Compose service resource to associate the environment variable with.</param>
+    /// <returns>A string representing the environment variable placeholder in Docker Compose syntax (e.g., <c>${ENV_VAR}</c>).</returns>
+    public static string AsEnvironmentPlaceHolder(this IManifestExpressionProvider manifestExpressionProvider, DockerComposeServiceResource dockerComposeService)
```

Comment:
Because I'm pedantic ... Placeholder not PlaceHolder

Reply:
... but approving nonetheless.

Reply:
Fixed